frontend/src/pages/devices/Detail.tsx

```diff
@@ -214,12 +231,31 @@ const Detail = () => {
             value={formatDistance(
               Date.parse(entity.lastRefreshedAt),
               new Date(),
-              { addSuffix: true }
+              { addSuffix: true },
             )}
             disabled={true}
           />
         </Form.Group>
 
+        <Form.Group>
+          <Form.Label>Controller</Form.Label>
+          <InputGroup className="mb-3">
+            <Form.Control
```

Comment:
Before wrapping the button in that div, try adding `aria-describedby="basic-addon2"` to the form group (like the [example](https://react-bootstrap.netlify.app/docs/forms/input-group/#button-addons)).

Reply:
e.g. if you copy the example directly, does that look right?

```jsx
      <InputGroup className="mb-3">
        <Form.Control
          placeholder="Recipient's username"
          aria-label="Recipient's username"
          aria-describedby="basic-addon2"
        />
        <Button variant="outline-secondary" id="button-addon2">
          Button
        </Button>
      </InputGroup>
```